packages/models-library/src/models_library/rpc/webserver/projects.py

```diff
@@ -2,12 +2,68 @@
 from typing import Annotated, TypeAlias
 from uuid import uuid4
 
-from models_library.projects import NodesDict, ProjectID
-from models_library.projects_nodes import Node
-from models_library.rpc_pagination import PageRpc
 from pydantic import BaseModel, ConfigDict, Field
 from pydantic.config import JsonDict
 
+from ...projects import NodesDict, ProjectID
+from ...projects_nodes import Node
+from ...rpc_pagination import PageRpc
+
+
+class MetadataFilterItem(BaseModel):
+    name: str
+    pattern: str
+
```

Comment:
minor: no limits here?

Reply:
no. In internal APIs i do not add limits. 
I do that only in external APIs for security. Once the guarantees are established there, the inner interfaces are not that restrictive.